piano/app/src/main/res/layout/fragment_black_key.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".BlackKeyFragment">
+
+    <Button
+        android:id="@+id/blackKey"
+        android:layout_width="30dp"
+        android:layout_height="133dp"
```

Comment:
Statisk høyde, burde være mer dynamisk for å kunne tilpasse seg høyde på ulike skjermstørrelser,

Reply:
Fikser dette i senere assignment siden UI ikke er ferdig